.circleci/config.yml

```diff
@@ -2,13 +2,13 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/rust:1.36.0
+      - image: circleci/rust:1.38.0
     steps:
       - checkout
       - run:
           name: setup
           command: |
-            rustup component add rustfmt --toolchain 1.36.0-x86_64-unknown-linux-gnu
+            rustup component add rustfmt
             rustup install nightly
```

Comment:
We install `nightly` but only use it to check `--no-default-features` <- is this right?

Reply:
Yea it's like this to ensure it will work with substrate which uses nightly to compile in no_std (aka. no-default-features)